contracts/Example2.sol

```diff
@@ -3,9 +3,11 @@ pragma solidity ^0.8.4;
 contract Example2 {
     uint public counter;
  
-    function incrementBy(uint[] memory arr) public {
+    function incrementBy(uint[] calldata arr) public {
```

Comment:
This can be an external function

Reply:
What's the point of changing it to external here?